<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Test10</class>
 <widget class="QMainWindow" name="Test10">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1242</width>
    <height>754</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Test10</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frmMainLeft_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>340</y>
      <width>211</width>
      <height>191</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="btnAnalyserAdd_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>93</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>btnAnalyserAdd</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSettings_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>93</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>btnSettings</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnReview_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>93</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>btnReview</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnExit_2">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>530</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>btnExit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnMenuHide">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>530</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>btnMenuHide</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>340</y>
      <width>511</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QFrame" name="frmMainLeft">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>230</y>
      <width>211</width>
      <height>191</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="btnAnalyserAdd">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>93</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>btnAnalyserAdd</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSettings">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>93</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>btnSettings</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnReview">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>93</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>btnReview</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnMenuShow_2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>170</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>btnMenuShow</string>
    </property>
   </widget>
   <widget class="QFrame" name="frmAddAnalisator">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>90</y>
      <width>621</width>
      <height>231</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="btnSaveAnal">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>100</y>
       <width>93</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>btnSaveAnal</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbxAddProducent">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>111</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="cbxAddModel">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>40</y>
       <width>101</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineAddSN">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>40</y>
       <width>113</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lblAddProducent">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>lblAddProducent</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblAddModel">
     <property name="geometry">
      <rect>
       <x>182</x>
       <y>10</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>lblAddModel</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblAddSN">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>10</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>lblAddSN</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnMenuShow">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>btnMenuShow</string>
    </property>
   </widget>
   <widget class="QFrame" name="frmAddAnalisator_2">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>200</y>
      <width>621</width>
      <height>231</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="btnSaveAnal_2">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>100</y>
       <width>93</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>btnSaveAnal</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbxAddProducent_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>111</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="cbxAddModel_2">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>40</y>
       <width>101</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineAddSN_2">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>40</y>
       <width>113</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lblAddProducent_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>lblAddProducent</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblAddModel_2">
     <property name="geometry">
      <rect>
       <x>182</x>
       <y>10</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>lblAddModel</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblAddSN_2">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>10</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>lblAddSN</string>
     </property>
    </widget>
   </widget>
   <widget class="QTableView" name="tableView_2">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>450</y>
      <width>511</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnExit">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>420</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>btnExit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnMenuHide_2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>640</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>btnMenuHide</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1242</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
